localization/sparse_mapping/include/sparse_mapping/sparse_map.h

```diff
@@ -272,15 +229,12 @@ struct SparseMap {
   std::vector<cv::Mat> cid_to_descriptor_map_;
   // generated on load
   std::vector<std::map<int, int> > cid_fid_to_pid_;
+  std::vector<std::map<int, int>> cid_to_matching_cid_counts_;
```

Comment:
If we're adding new fields to the map, ideally we should also log their memory usage in our memory debug statistics. (Not sure if you were aware there's per-field code for that.)

Reply:
Oh actually I wasn't aware of that, which file(s) should I edit?

Reply:
(found it, looks like it's still in PR, I'll update the pr)